imageio/v3.py

```diff
@@ -0,0 +1,6 @@
+from .core.v3_api import imopen
+from .core.v3_api import imread
+from .core.v3_api import imwrite
+from .core.v3_api import imiter
+
+__all__ = ["imopen", "imread", "imwrite", "imiter"]
```

Comment:
Why not rename `v3_api` to `v3` and include the `__all__` there :)

Reply:
Sure, we could do that. Alternatively, I can also move the functions themself from `core.v3_api` into `v3`. Which organizational structure do you think makes more sense?

Reply:
The end-result should be the same, right? :)

Reply:
Looking at this now (about 1 month later), I think it actually makes sense to keep them separate or move `imopen` back into its own file.

Both the v2 and the v3 functions use `imopen` internally, so - in the light of the results of #725 - I am rethinking how to structure the library to accommodate this long term. Haven't settled for anything specific yet though.

Reply:
I ended up moving the API functions into v3.py ; however, I have reintroduced `imopen.py` since it is a shared dependency between the v2 API and the v3 API. This now works nicely without any odd dependency problems.